### PR TITLE
ngo upgrade

### PR DESCRIPTION
**Pull request by IgorMinar**

### CODE REVIEW

File: aio/package.json [hunk 1]
```diff
@@ -34,7 +34,7 @@
     "generate-zips": "node ./tools/example-zipper/generateZips",
     "sw-manifest": "ngu-sw-manifest --dist dist --in ngsw-manifest.json --out dist/ngsw-manifest.json",
     "sw-copy": "cp node_modules/@angular/service-worker/bundles/worker-basic.min.js dist/",
-    "postinstall": "node --eval \"const fs=require('fs'), sh=require('shelljs'), PATCH_LOCK='node_modules/@angular/cli/models/webpack-configs/.patched'; if (!fs.existsSync(PATCH_LOCK)) { sh.touch(PATCH_LOCK); sh.exec('patch -p0 -i tools/cli-patches/ngo-loader.patch && patch -p0 -i node_modules/purify/angular-cli.patch'); }\"",
+    "postinstall": "node --eval \"const fs=require('fs'), sh=require('shelljs'), PATCH_LOCK='node_modules/@angular/cli/models/webpack-configs/.patched'; if (!fs.existsSync(PATCH_LOCK)) { sh.touch(PATCH_LOCK); sh.exec('patch -p0 -i tools/cli-patches/ngo-loader.patch && patch -p0 -i node_modules/purify/angular-cli.patch && patch -p0 -i tools/cli-patches/uglify-config.patch'); }\"",
```
Comment (petebacondarwin) [inline aio/package.json]: @IgorMinar - do you think we could move this into a script? It is becoming more and more difficult to follow what it is doing.

Comment (IgorMinar) [inline aio/package.json]: ok. I'll redo it.

**Comment by IgorMinar** [comment]
@gkalpak this PR contains a commit with the new firebase token.

**Review by petebacondarwin — APPROVED** [review]

File: .travis.yml [hunk 2]
```diff
@@ -34,9 +34,9 @@ env:
     # This is needed for the e2e Travis matrix task to publish packages to github for continuous packages delivery.
     - secure: "aCdHveZuY8AT4Jr1JoJB4LxZsnGWRe/KseZh1YXYe5UtufFCtTVHvUcLn0j2aLBF0KpdyS+hWf0i4np9jthKu2xPKriefoPgCMpisYeC0MFkwbmv+XlgkUbgkgVZMGiVyX7DCYXVahxIoOUjVMEDCbNiHTIrfEuyq24U3ok2tHc="
     # FIREBASE_TOKEN
-    # This is needed for publishing builds to the "aio-staging" firebase site.
-    # TODO(i): the token was generated using the iminar@google account, we should switch to a shared/role-base account.
-    - secure: "MPx3UM77o5IlhT75PKHL0FXoB5tSXDc3vnCXCd1sRy4XUTZ9vjcV6nNuyqEf+SOw659bGbC1FI4mACGx1Q+z7MQDR85b1mcA9uSgHDkh+IR82CnCVdaX9d1RXafdJIArahxfmorbiiPPLyPIKggo7ituRm+2c+iraoCkE/pXxYg="
+    # This is needed for publishing builds to the "aio-staging" and "angular-io" firebase projects.
+    # This token was generated using the aio-deploy@angular.io account using `firebase login:ci` and password from valentine
+    - secure: "L5CyQmpwWtoR4Qi4xlWQh/cL1M6ZeJL4W4QAr4HdKFMgYt9h+Whqkymyh2NxwmCbPvWa7yUd+OiLQUDCY7L2VIg16hTwoe2CgYDyQA0BEwLzxtRrJXl93TfwMlrUx5JSIzAccD6D4sjtz8kSFMomK2Nls33xOXOukwyhVMjd0Cg="
```
Comment (petebacondarwin) [inline .travis.yml]: Publishing this to Github doesn't seem very "secure"

Comment (gkalpak) [inline .travis.yml]: This is just the public part. It is [secure enough][1] :smiley: 

[1]: https://docs.travis-ci.com/user/environment-variables/#Defining-encrypted-variables-in-.travis.yml

**Pull request merged**